### DISCUSSION TITLE
Default parameters to swtpm_setup from configuration file

### DISCUSSION THREAD

**Discussion by deesceha** [stefanberger/swtpm#871] [Q&A] [answered]
Is it possible to set parameter default values to swtpm_setup in a config file? For example, I would like to set the RSA key size (CLI parameter "--rsa-keysize").

**Comment by stefanberger** [comment]
`swtpm_setup.conf` exists today but it does not have support for this configuration option.

**Comment by stefanberger** [comment]
PR https://github.com/stefanberger/swtpm/pull/874 should help resolve this issue.

**Reply by stefanberger** [reply, marked as answer]
swtpm_setup.conf of upcoming v0.10 allows a user to specify the rsa_keysize with the same values as on the command line.
